cli/cmd/top.go

```diff
@@ -630,6 +654,25 @@ func (t *topTable) renderHeaders() {
 	}
 }
 
+func (t *topTable) tableWidthCalc() int {
+	tablewidth := 0
+	for i, col := range t.columns {
+		t.columns[i].width = runewidth.StringWidth(col.header)
+		for _, row := range t.rows {
+			cellWidth := runewidth.StringWidth(col.value(row))
+			if cellWidth > t.columns[i].width {
+				t.columns[i].width = cellWidth
+			}
+		}
+		if col.flexible {
```

Comment:
why do we only add the column spacing when the column is flexible?

I'm comparing this to the code in `renderBody` which increments `x` by both the column width and the column spacing for each column.  Wouldn't that be the right way to calculate the total table width?

Reply:
Sorry. My mistake. I have corrected it.